x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/overview/overview.tsx

```diff
@@ -36,6 +38,10 @@ import { isPackageUnverified } from '../../../../../../../services';
 import type { PackageInfo, RegistryPolicyTemplate } from '../../../../../types';
 import { SideBarColumn } from '../../../components/side_bar_column';
 
+import type { FleetStartServices } from '../../../../../../../plugin';
+
+import { AVCResultsBanner2024 } from './avc_banner/avc_results_banner_2024';
```

Comment:
If you choose to import this from the `security_solution` plugin you would probably need to add `@kbn/security-solution-plugin"` to `kbn_references` list in `x-pack/plugins/fleet/tsconfig.json`